### PR TITLE
tests: add logic to not fail when trying to send the logs to s3

### PR DESCRIPTION
**Pull request by cpanato**
followup of: https://github.com/coreos/tectonic-installer/pull/2816

master will fail now because the env variable is not defined. was my fault. I thought we send logs only in the end. this fixes the issue.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -302,6 +302,7 @@ pipeline {
         TF_VAR_tectonic_container_images = "${params.hyperkube_image}"
         TF_VAR_tectonic_kubelet_debug_config = "--minimum-container-ttl-duration=8h --maximum-dead-containers-per-container=9999 --maximum-dead-containers=9999"
         GOOGLE_PROJECT = "tectonic-installer"
+        LOGSTASH_BUCKET= "log-analyzer-tectonic-installer"
```
Comment (mxinden) [inline Jenkinsfile]: This makes `LOGSTASH_BUCKET` defined twice, right?

Comment (cpanato) [reply Jenkinsfile]: forgot to remove, doing that

Comment (cpanato) [reply Jenkinsfile]: done @mxinden

Comment (cpanato) [reply Jenkinsfile]: already pushed the changes. remove the variable from the post operation and keep that in high level

**Comment by cpanato** [comment]
retest this please. aws only

**Pull request merged**